pkg/nodeconfig/nodeconfig.go

```diff
@@ -130,6 +131,48 @@ func discoverKubeAPIServerEndpoint() (string, error) {
 	return host.Status.APIServerInternalURL, nil
 }
 
+// getWICDCredentials returns the CA cert and access token associated with the WICD service account
+func getWICDCredentials() (*windows.Authentication, error) {
+	cfg, err := crclientcfg.GetConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get config to talk to kubernetes api server")
+	}
+	client, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get client from the given config")
+	}
+
+	operatorNamespaceVar := "WATCH_NAMESPACE"
```

Comment:
Isn't there an `OPERATOR_NAMESPACE` variable also? If yes, wouldn't that be more appropriate?

Reply:
I don't think `OPERATOR_NAMESPACE` gets set, closest thing I see is `OPERATOR_NAME`.